client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -0,0 +1,350 @@
+ */
+import { useState } from '@wordpress/element';
+import {
+	Button,
+	Card,
+	CheckboxControl,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { Link } from '@woocommerce/components';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { Icon, chevronDown, chevronUp } from '@wordpress/icons';
+import interpolateComponents from 'interpolate-components';
+import { pluginNames } from '@woocommerce/data';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import { AppIllustration } from '../app-illustration';
+import './style.scss';
+
+const generatePluginDescriptionWithLink = ( description, productName ) => {
+	return interpolateComponents( {
+		mixedString: description,
+		components: {
+			link: (
+				<Link
+					type="external"
+					target="_blank"
+					className="woocommerce-admin__business-details__selective-extensions-bundle__link"
+					href={ `https://woocommerce.com/products/${ productName }` }
+				/>
+			),
+		},
+	} );
+};
+
+const installableExtensions = [
+	{
+		title: __( 'Get the basics', 'woocommerce-admin' ),
+		plugins: [
+			{
+				slug: 'woocommerce-payments',
+				description: generatePluginDescriptionWithLink(
+					__(
+						'Accept credit cards with {{link}}WooCommerce Payments{{/link}}',
+						'woocommerce-admin'
+					),
+					'woocommerce-payments'
+				),
+			},
+			{
+				slug: 'woocommerce-services',
+				description: generatePluginDescriptionWithLink(
+					__(
+						'Print shipping labels with {{link}}WooCommerce Shipping{{/link}}',
+						'woocommerce-admin'
+					),
+					'shipping'
+				),
+			},
+			{
+				slug: 'jetpack',
+				description: generatePluginDescriptionWithLink(
+					__(
+						'Enhance speed and security with {{link}}Jetpack{{/link}}',
+						'woocommerce-admin'
+					),
+					'jetpack'
+				),
+			},
+		],
+	},
+	{
+		title: 'Grow your store',
+		plugins: [
+			{
+				slug: 'facebook-for-woocommerce',
+				description: generatePluginDescriptionWithLink(
+					__(
+						'Market on {{link}}Facebook{{/link}}',
+						'woocommerce-admin'
+					),
+					'facebook'
+				),
+			},
+			{
+				slug: 'kliken-marketing-for-google',
+				description: generatePluginDescriptionWithLink(
+					__(
+						'Drive sales with {{link}}Google Ads{{/link}}',
+						'woocommerce-admin'
+					),
+					'google-ads-and-marketing'
+				),
+			},
+			{
+				slug: 'mailchimp-for-woocommerce',
+				description: generatePluginDescriptionWithLink(
+					__(
+						'Contact customers with {{link}}Mailchimp{{/link}}',
+						'woocommerce-admin'
+					),
+					'mailchimp-for-woocommerce'
+				),
+			},
+			{
+				slug: 'creative-mail-by-constant-contact',
+				description: generatePluginDescriptionWithLink(
+					__(
+						'Reach new customers with {{link}}Creative Mail{{/link}}',
+						'woocommerce-admin'
+					),
+					'creative-mail-for-woocommerce'
+				),
+			},
+		],
+	},
+];
+
+const initialValues = installableExtensions.reduce(
```

Comment:
The initial form values are derived from the available extensions, this is just more flexible and going to save some hassle for the next person to add an extension to the list. They can just add items to installable extensions.